Assets/Scripts/Cards/Collection/ShieldCard.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections;
```

Comment:
small nitpick but can we drop the Card suffix on these scripts and name them the same as the `name` value on the actual card?